actor/builtin/paymentbroker/paymentbroker_test.go

```diff
@@ -753,10 +755,11 @@ func (sys *system) applySignatureMessage(target address.Address, amtInt uint64,
 	require := require.New(sys.t)
 
 	amt := types.NewAttoFILFromFIL(amtInt)
-	signature, err := sys.Signature(amt, validAt)
+	signature, err := sys.Signature(amt, validAt, nil)
 	require.NoError(err)
 
-	pdata := core.MustConvertParams(sys.payer, sys.channelID, amt, validAt, signature)
+	var predicate *types.Predicate
+	pdata := core.MustConvertParams(sys.payer, sys.channelID, amt, validAt, predicate, signature)
```

Comment:
Blocking: some test somewhere here must verify that non-empty condition is correctly encoded in the signed voucher.

Reply:
I've added tests to ensure vouchers are generated correctly with conditions, and that voucher signing includes the condition.